test/lib.reviews.js

```diff
@@ -8,6 +8,7 @@ function assertValid (review) {
   assert.isString(review.id);
   assert(review.id);
   assert.isString(review.userName);
+  assert('userImage' in review);
   assert(review.userName);
```

Comment:
I'm not confident, we'd have to see what gplay does when there's no image (I don't have a computer available to do it at the moment).

If there isn't always a URL we can wrap the assert on an if. Ideally the test should break whenever the selector ceases to work.